benchmark/README.md

```diff
@@ -0,0 +1,37 @@
+# Keras Benchmarks
+
+## Overview
+The benchmakr module aims to provide a performance comparison on different Keras backends using various models and dataset.
+Currently supported backends: TensorFlow, MXNet 
+
+## Setup
+To switch between different backends refer to this documentation:
+
+We provide benchmark scripts to run on ImageNet DataSet and Synthetic Dataset(randomly generated) 
+### ImnageNet Dataset
+There are two training mode: train_on_batch and fit_generator, we recommend train_on_batch since it's more efficient on multi_gpu.
+Need to provide training mode, number of gpus and path to imagenet dataset.
+
+Example usage:
+
+`python benchmark_imagenet_resnet.py --train_mode train_on_batch --gpus 4 --data_path home/ubuntu/imagenet/train/`
```

Comment:
Any links to know where to download Imagenet dataset? Size and bit more details on the Imagenet dataset?